docs/wiki/[Examples]-Deploy-Management-Resources-With-Custom-Settings.md

```diff
@@ -101,7 +102,7 @@ variable "security_alerts_email_address" {
 
 variable "management_resources_location" {
   type    = string
-  default = "uksouth"
+  default = "eastus"
```

Comment:
Just an FYI, I am reverting this to `uksouth` as `eastus` is the global module default for this and the point of the custom demo is to set a different location.